README.md

```diff
@@ -0,0 +1,36 @@
+# Terraform Static Website Infrastructure
+
+This repository presents the use of a module with an infrastructure for hosting a React application with AWS S3 and AWS CloudFront with the possibility of creating domain aliases with AWS Route53 and generating a certificate with Amazon Certificate Manager.
+
+## Requirements
+
+| Name | Version |
+|---|---|
+| terraform  | 1.2.3 |
+| aws-cli    | 2.4.2 |
+
+## Providers
+
+| Name | Version |
+|---|---|
+| aws  | >= 4.19.0 |

```

Comment:
While it doesn't matter for the GitHub Markdown parser --- or nearly any other parser --- it might be interesting to align these tables, even if just to make it look prettier when editing. There are IDE plugins to easily align Markdown tables.

On top of that, how about centering the `Version` columns?

```suggestion
| Name      | Version |
|:----------|:-------:|
| terraform | 1.2.3   |
| aws-cli   | 2.4.2   |

## Providers

| Name | Version   |
|:-----|:---------:|
| aws  | >= 4.19.0 |

```